### PR TITLE
docs(socialaccount): add config example for Authelia

### PR DESCRIPTION
**Pull request by noantiq**
# Submitting Pull Requests

## General

 - [ ] Make sure you use [semantic commit messages](https://seesparkbox.com/foundry/semantic_commit_messages).
       Examples: `"fix(google): Fixed foobar bug"`, `"feat(accounts): Added foobar feature"`.
 - [ ] All Python code must formatted using Black, and clean from pep8 and isort issues.
 - [ ] JavaScript code should adhere to [StandardJS](https://standardjs.com).
 - [ ] If your changes are significant, please update `ChangeLog.rst`.
 - [ ] If your change is substantial, feel free to add yourself to `AUTHORS`.

 ## Provider Specifics

 In case you add a new provider:

- [ ] Make sure unit tests are available.
- [ ] Add an entry of your provider in `test_settings.py::INSTALLED_APPS` and `docs/installation.rst::INSTALLED_APPS`.
- [ ] Add documentation to `docs/providers/<provider name>.rst` and `docs/providers/index.rst` Provider Specifics toctree.
- [ ] Add an entry to the list of supported providers over at `docs/overview.rst`.

### CODE REVIEW

**Comment by noantiq** [comment]
I struggled a bit to figure out the correct config values to use on each site when setting up https://github.com/paperless-ngx/paperless-ngx with https://github.com/authelia/authelia using OIDC. Hopefully this can be helpful for someone.

File: docs/socialaccount/providers/authelia.rst [hunk 1]
```diff
@@ -0,0 +1,28 @@
+Authelia
+--------
+
+At the time of writing, `Authelia <https://www.authelia.com/>`__ supports OpenID Connect (OIDC) as a beta feature. Detailed information about the available configuration options can be found on their `website <https://www.authelia.com/configuration/identity-providers/open-id-connect/>`__.
+
+As documented at https://www.authelia.com/integration/openid-connect/introduction/ the Well Known Discovery Endpoint that can be used as the server url is ``https://auth.example.com/.well-known/openid-configuration`` where ``https://auth.example.com/`` should be replaced by the specific url of your instance of Authelia.
+
+An example configuration for authelia would look like this:
```
Comment (pennersr) [inline docs/socialaccount/providers/authelia.rst:8]:
```suggestion
An example configuration for authelia would look like this::
```

File: docs/socialaccount/providers/authelia.rst [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+::
```
Comment (pennersr) [inline docs/socialaccount/providers/authelia.rst:10]:
```suggestion
```

**Review by pennersr — CHANGES_REQUESTED** [review]

**Comment by pennersr** [comment]
Question, there is this open ticket:

https://github.com/pennersr/django-allauth/issues/3631

Is Authelia working for you? Why isn't that ticket causing trouble for you?

**Comment by noantiq** [comment]
Yes, it’s working for me. The stable version as well as the newest beta.

**Pull request merged**